wordpress-mobile/WordPress-iOS
# Comments: Show a spinner when refreshing comment lists.

Closes #16480
This PR causes the CommentsViewController's tableView's refreshControl to become visible as the list is refreshed when switching filters. 
Example:
![filter-spinner](https://user-images.githubusercontent.com/1435271/118716699-bf9e8780-b7ea-11eb-9dd0-2a477d745521.gif)

To test:
View the comments list.  Tap between different filters.  Confirm a spinner is shown while the list is loading. Confirm the animation is smooth and not jarring. 

@ScoutHarris Game for another? 

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a
PR submission checklist:
